se2_planning/src/OmplReedsSheppPlanner.cpp

```diff
@@ -63,6 +69,35 @@ void OmplReedsSheppPlanner::setStateSpaceBoundaries() {
   bounds_->high[0] = parameters_.xUpperBound_;
   bounds_->high[1] = parameters_.yUpperBound_;
   stateSpace_->as<ompl::base::SE2StateSpace>()->setBounds(*bounds_);
+  //  std::cout << "OmplReedsSheppPlanner: Set state space bounds: x = [" << bounds_->low[0] << ", " << bounds_->high[0] << "], y=["
+  //            << bounds_->low[1] << ", " << bounds_->high[1] << "]" << std::endl;
+}
+const ompl::base::RealVectorBounds& OmplReedsSheppPlanner::getStateSpaceBoundaries() {
+  return stateSpace_->as<ompl::base::SE2StateSpace>()->getBounds();
+}
+
+void OmplReedsSheppPlanner::updateStateSpaceBounds(const ompl::base::RealVectorBounds& bounds) {
+  bounds_->low[0] = bounds.low[0];
+  bounds_->low[1] = bounds.low[1];
+  bounds_->high[0] = bounds.high[0];
+  bounds_->high[1] = bounds.high[1];
+  parameters_.xLowerBound_ = bounds_->low[0];
+  parameters_.yLowerBound_ = bounds_->low[1];
+  parameters_.xUpperBound_ = bounds_->high[0];
+  parameters_.yUpperBound_ = bounds_->high[1];
+  stateSpace_->as<ompl::base::SE2StateSpace>()->setBounds(*bounds_);
+  //  std::cout << "OmplReedsSheppPlanner: Update state space bounds: x = [" << bounds_->low[0] << ", " << bounds_->high[0] << "], y=["
+  //            << bounds_->low[1] << ", " << bounds_->high[1] << "]" << std::endl;
+}
+
+bool OmplReedsSheppPlanner::satisfiesStateSpaceBounds(const se2_planning::ReedsSheppState& state) {
+  ompl::base::ScopedStatePtr stateOmpl(std::make_shared<ompl::base::ScopedState<> >(stateSpace_));
+  auto s = ((*stateOmpl)())->as<ompl::base::SE2StateSpace::StateType>();
+  auto rsState = state.as<ReedsSheppState>();
```

Comment:
make this method const

Reply:
Done in c8ad75dfd72956b0eca0707dfb0dbfb1532d5eed.